var/spack/repos/builtin/packages/openfoam/package.py

```diff
@@ -351,7 +352,7 @@ class Openfoam(Package):
     )
```

Comment:
Apologies but our flake rules have changed and flake now thinks this is problematic. IMO we should figure out how to allow this since I think it looks better than what flake would be ok with (e.g. move this paren to the end of the previous line like `3e5d61')` but in the meantime if you are able to update this it will get it past our CI.

Reply:
I think you can either do what you describe *or* move the first line down one line and indent 4 and have a closing paren like this, but you can’t mix the two.

Reply:
If the flake8 rules have changed, are they not reflected in the repo somehow? If not, getting packages to be accepted becomes even more hit-and-miss.
If I fetch develop (right now - commit bc145b2b) and rebase my branch onto it and run things:
```
$ spack flake8 openfoam/package.py
=======================================================
flake8: running flake8 code checks on spack.

Modified files:
  var/spack/repos/builtin/packages/openfoam/package.py
=======================================================

Flake8 checks were clean.
```

I have not knowingly created or changed any flake8 config files anywhere in the repo hierarchy or home directory so I would expect that running the above should reflect what the CI accepts.

The _offending_ indentation with a closing ')' at the same level as the initial non-whitespace line seems to be in accordance with https://www.python.org/dev/peps/pep-0008/#indentation

> or it may be lined up under the first character of the line that starts the multiline construct, as in:

Reply:
What version of flake8 are you using?

Reply:
python2-flake8-3.7.7-lp151.1.1.noarch

Reply:
I think Travis CI is using flake8 3.8.2, so if you want to be sure, I would use the same version. 

Short term answer: all PRs must pass flake8 or we can't merge them. Long term answer: we're thinking about switching to black in the future so that no one has to worry about silly things like this.